kotlin/scene-layer/src/main/java/com/esri/arcgisruntime/sample/scenelayer/MainActivity.kt

```diff
@@ -37,7 +37,7 @@ class MainActivity : AppCompatActivity() {
     val camera = Camera(48.378, -4.494, 200.0, 345.0, 65.0, 0.0)
 
     // create a scene
-    val scene = ArcGISScene().apply {
+    val brestBuildings = ArcGISScene().apply {
```

Comment:
I think `brestBuildingsScene` would be a better variable name: again just to make it clear to the users what object this variable is referencing. As the name currently stands, it could be a number of things from our API.